ChangeLog.md

```diff
@@ -1,5 +1,175 @@
 # Changelog for haskell-language-server
 
+## 1.0.0
+
+This is the celebratory release of Haskell Language Server 1.0.0!
+This release includes a lot of internal changes, bug fixes, and performance improvements, thanks to all our contributors.
+Among others,
+- We added the support for GHC 8.10.4, and removed the support for GHC 8.10.1
+    Afterward, we will support upmost 3 patch versions for each minor version of GHC, if no special situation happens.
+- Now Eval plugin uses the parsing mechanism of GHC and becomes more stable.
+- We supports a code action to hide shadowed identifiers from import.
```

Comment:
Can we refer to also an import disambiguation?

Reply:
Sure, I will add them when I reflect comments.